### ISSUE TITLE
You stumbled upon a bug - renderPage function when migrating from version 0.4.150 to 0.4.151

### ISSUE THREAD

**Issue by gasparearmato6991** [vikejs/vike#1404]
### Description

Hello! We just stumbled on a bug upgrading Vike to version 0.4.151, it's working fine with the 0.4.150 but it's not with all the newer versions (we also tried with the latest one, the 0.4.153)

```
Error: [vike@0.4.151][Bug] You stumbled upon a bug in Vike's source code. Go to https://github.com/vikejs/vike/issues/new and copy-paste this error; a maintainer will fix the bug (usually under 24 hours).
    at file:///Users/username/Desktop/org/repos/repo/node_modules/vike/dist/esm/node/runtime/globalContext/loadImportBuild.js:16:13
    at autoRetry (file:///Users/username/Desktop/org/repos/repo/node_modules/vike/dist/esm/utils/autoRetry.js:9:19)
    at async loadImportBuild (file:///Users/username/Desktop/org/repos/repo/node_modules/vike/dist/esm/node/runtime/globalContext/loadImportBuild.js:15:9)
    at async initGlobalContext (file:///Users/username/Desktop/org/repos/repo/node_modules/vike/dist/esm/node/runtime/globalContext.js:75:30)
    at async renderPageAndPrepare (file:///Users/username/Desktop/org/repos/repo/node_modules/vike/dist/esm/node/runtime/renderPage.js:66:9)
    at async renderPage_wrapper (file:///Users/username/Desktop/org/repos/repo/node_modules/vike/dist/esm/node/runtime/renderPage.js:26:24)
    at async renderPage (file:///Users/username/Desktop/org/repos/repo/node_modules/vike/dist/esm/node/runtime/renderPage.js:46:50)
    at async file:///Users/username/Desktop/org/repos/repo/dist/index.js:220:27
```

And our `index.ts` looks like that:
```
import express from "express";
import {renderPage} from "vike/server";

async function startServer(): Promise<void> {
    const app = express()

    app.get("*", async (req, res, next) => {

        const pageContextInit = {
            urlOriginal: req.originalUrl,
            environment: {
                CUSTOM_ENV_VAR: "VALUE",
            }
        };
        const pageContext = await renderPage(pageContextInit);
    })
}

startServer();

```

We are using Vike with Svelte if it can be useful.

**Comment by brillout** [comment]
Do you get this with 0.4.153?

**Comment by gasparearmato6991** [comment]
No, the error is appearing when we switch from 0.4.150 to 0.4.151 but even trying with a newer version (e.g. 0.4.152 or 0.4.153 it does not work)

**Comment by brillout** [comment]
If you increase the timeout from `2000` to `20000` at line `17` of the file `/Users/username/Desktop/org/repos/repo/node_modules/vike/dist/esm/node/runtime/globalContext/loadImportBuild.js`, do you get the same error or another error?

**Comment by gasparearmato6991** [comment]
Unfortunately no, it just start retrying more and more times until in reaches the `20s` timeout

**Comment by brillout** [comment]
That's quite unexpected. I'll need a reproduction, ideally a minimal one although it doesn't have to.

**Comment by brillout** [comment]
Fix pre-released in `0.4.153-commit-e631897 `.

**Comment by brillout** [comment]
Also, would your company be up for [sponsoring](https://github.com/vikejs/vike/issues/1350)?
